internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -109,7 +122,15 @@ func (r ReconcilePersistentVolume) reconcilePV(obj runtime.Object) error {
 		volumeHandler := pv.Spec.CSI.VolumeHandle
 		secretName := ""
 		secretNamespace := ""
-
+		// check static volume
+		static, err := checkStaticVolume(pv.Spec.CSI.VolumeAttributes["staticVolume"])
```

Comment:
It is safer to pass `pv` or one of its components to `checkStaticVolume()`, and verify in the function that the `"staticVolume"` key does exist.